server/routes/index.ts

```diff
@@ -11,10 +11,14 @@ import spikeRoutes from './spikes'
 import errorMessageMiddleware from '../middleware/errorMessageMiddleware'
 import successMessageMiddleware from '../middleware/successMessageMiddleware'
 import timeNowMiddleware from '../middleware/timeNowMiddleware'
+import routeAuthMiddleware from '../middleware/routeAuthMiddleware'
 import appointmentRoutes from './appointments'
 
 export default function routes(services: Services): Router {
   const router = Router({ mergeParams: true })
+
+  router.use(routeAuthMiddleware())
+
```

Comment:
This is the choice - middleware on every request, or limited to specific route handlers. Good for now.